admin/model_cluster_free_auto_scaling.go

```diff
@@ -4,8 +4,7 @@ package admin
 
 // ClusterFreeAutoScaling Range of instance sizes to which your cluster can scale.
 type ClusterFreeAutoScaling struct {
-	// Collection of settings that configures how a cluster might scale its cluster tier and whether the cluster can scale down.
-	Compute *string `json:"compute,omitempty"`
+	Compute *FreeComputeAutoScalingRules `json:"compute,omitempty"`
```

Comment:
change is related to the spec correction for tenant upgrade endpoint: CLOUDP-278225